src/docs/product/session-replay/protecting-user-privacy.mdx

```diff
@@ -18,12 +27,38 @@ Alternatively, you can turn off the default privacy configuration, which will ca
 
 To learn more, including available configuration options, see [Session Replay Privacy](/platforms/javascript/session-replay/privacy/).
 
-## User Input
+### User Input
 
 In addition to HTML content, Session Replay records key user interactions like mouse movements, mouse clicks, and keyboard presses. By default, all user keypresses are redacted and replaced with asterisks. Captured mouse movements are limited to source and target destinations (for example, where the mouse came from and where it went, not the exact path). A low-fidelity approximation of this movement is recreated during playback.
 
-## Breadcrumbs and other Metadata
+### Breadcrumbs and other Metadata
 
 To assist in debugging, Session Replay captures additional metadata like console.log entries, network requests, browser, operating system, and more. By default, Sentry scans this data on the server and automatically removes potentially sensitive strings like credit card numbers, social security numbers, API tokens, and more. This same process is applied to other Sentry event types like Errors and Transactions.
 
-To learn more see [PII and Data Scrubbing](/product/relay/pii-and-data-scrubbing/).
+To learn more see [Server-Side Data Scrubbing](/product/data-management-settings/scrubbing/server-side-scrubbing/).
+
+## Legal and Privacy FAQ
+
+**Q: Can I use Session Replay and comply with the GDPR and CCPA?**
+
+A: Yes, we offer a [Data Processing Addendum (DPA)](https://sentry.io/legal/dpa/) to cover GDPR and CCPA requirements for any personal data sent to Sentry, including via our Session Replay product. You can enter into our DPA by following the instructions [here](https://sentry.io/trust/privacy/).
+
+**Q: I have an existing DPA with Sentry, does it apply to Session Replay?**
+
+A: Yes, see above. The data processing and security commitments in our DPA apply to the personal data you send to the Sentry service (including via our Session Replay product).
+
+**Q: I have an existing Business Associate Addendum (BAA) with Sentry, does it apply to Session Replay?**
+
+A: Yes, [our BAA](https://sentry.io/legal/baa/) covers HIPAA requirements for any protected health information (PHI) sent to the Sentry service (including if you elect to do so via our Session Replay product).
```

Comment:
Does the text inside the parenthesis need some extra commas thrown in too?

Reply:
Mmm, seems okay to me. Could you elaborate?